src/lib/FileReadWriteDataSource.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
```

Comment:
This file should be under `./datasource` folder

Reply:
done